docs/Components/AttachedComponent.md

```diff
@@ -59,4 +59,13 @@ print(math.degrees(aileron.angle))
 plane.angle = math.radians(5)
 print(math.degrees(aileron.angle))
 >>> 14.99999999999
+```
+## position property
+This property returns a position vector of the child component considering the position of the parent component
```

Comment:
"This property returns the position vector of the component's origin in relation to its parent's origin"